_docs/concepts/security/mutual-tls.md

```diff
@@ -33,9 +33,9 @@ on both Kubernetes containers and VM/bare-metal machines.
     caption='Istio Security Architecture'
     %}
 
-As illustrated in the diagram, Istio Security leverages secret volume mount to deliver keys/certs from Istio Citadel to Kubernetes containers. For services running on
+As illustrated in the diagram, Istio Security leverages secret volume mount to deliver keys/certs from Citadel to Kubernetes containers. For services running on
```

Comment:
Here and elsewhere, I don't think we want to say "Istio Security" does this or "Istio Security" does that. Just like we don't say "Istio Traffic Management" routes traffic or Istio Policies and Telemetry enforces policies.

In all these cases, let's just say "Istio"

So:

Istio enforces security by leveraging Citadel.

Reply:
Done.